src/community/china/_os-settings.md

```diff
@@ -0,0 +1,106 @@
+{% assign id =  include.os | downcase -%}
+
+{% if id == 'windows' -%}
+{% assign shell = 'Powershell' -%}
+{% assign prompt = 'C:\>' -%}
+{% assign envvarset = 'C:\> $env:' -%}
+{% assign setpath = 'C:\> $env:PATH = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"' -%}
+{% assign comtoset = '$env:' -%}
+{% assign unzip='Extract-Archive' %}
+{% capture permaddexample -%}
+$newPath = $pwd.PATH + "/flutter/bin",$env:PATH -join ";"
+[System.Environment]::SetEnvironmentVariable('Path',$newPath,User)
+[System.Environment]::SetEnvironmentVariable('PUB_HOSTED_URL','https://pub.flutter-io.cn',User)
+[System.Environment]::SetEnvironmentVariable('FLUTTER_STORAGE_BASE_URL','https://storage.flutter-io.cn',User)
+{% endcapture -%}
+{% else -%}
+{% assign shell = 'your terminal' -%}
+{% assign prompt = '$' -%}
+{% assign envvarset = '$ export ' -%}
+{% assign setpath = '$ export PATH="$PWD/flutter/bin:$PATH"' -%}
+{% assign comtoset = 'export' -%}
+{% assign unzip='unzip' %}
```

Comment:
To be consistent with the other commands and fix syntax highlighting, should this start with a `$`?

```suggestion
{% assign unzip='$ unzip' %}
```